examples/plugins/README.rst

```diff
@@ -14,3 +14,9 @@ And to remove them on a development environment, you may run, at the
 same directory::
 
  $ python setup.py develop --uninstall --user
+
+Second way to add to install plugin is to use Makefile's link target,
+Simply add symlink of target plugin to parent avocado directory and
+do make link::
+ $ ln -s examples/plugins/job-pre-post/mail ../mail-plugin
+ $ make link
```

Comment:
Nice, this is more convenient. I remember @clebergnu talked about managing those extra plugins somehow so let's get his opinion whether it clashes with it or does not. I do like it :+1: